test/connectors/addressLookup/AddressLookupConnectorSpec.scala

```diff
@@ -0,0 +1,185 @@
+/*
+ * Copyright 2020 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package connectors.addressLookup
+
+import assets.constants.ConfirmedAddressConstants
+import base.SpecBase
+import com.github.tomakehurst.wiremock.client.WireMock._
+import config.FrontendAppConfig
+import connectors.WireMockHelper
+import connectors.addressLookup.httpParsers.AddressLookupInitializationHttpParser.AddressLookupOnRamp
+import connectors.addressLookup.httpParsers.{AddressMalformed, AddressNotFound, DefaultedUnexpectedFailure, NoLocationHeaderReturned}
+import org.mockito.Mockito.when
+import org.scalatestplus.mockito.MockitoSugar
+import play.api.http.HeaderNames
+import play.api.http.Status._
+import play.api.libs.json.Json
+import uk.gov.hmrc.play.bootstrap.http.HttpClient
+
+
+class AddressLookupConnectorSpec extends SpecBase with WireMockHelper with MockitoSugar {
+
+  lazy val mockFrontendAppConfig: FrontendAppConfig = mock[FrontendAppConfig]
+
+
+  "AddressLookupConnector" should {
+    val httpClient : HttpClient = injector.instanceOf[HttpClient]
+    lazy val addressLookupConnector : AddressLookupConnector = new AddressLookupConnector(httpClient, mockFrontendAppConfig)
+
+    when(mockFrontendAppConfig.addressLookupFrontend) thenReturn getUrl
+    when(mockFrontendAppConfig.host) thenReturn "http://localhost:9457"
+    when(mockFrontendAppConfig.feedbackUrlAddressLookup) thenReturn "http://localhost:9250/contact/beta-feedback?service=charities"
+    when(mockFrontendAppConfig.contactFormServiceIdentifier) thenReturn "charities"
+    when(mockFrontendAppConfig.timeout) thenReturn 900
+    when(mockFrontendAppConfig.retrieveAddressUrl) thenReturn (getUrl + "/api/confirmed")
+
+    "For the .initialise() method" should {
```

Comment:
In some areas of the application we are using initialise, but in others we are using initialize. It doesn't affect the functionality of the application as they are defined separately, but it would be good to make them the same spelling for consistency.